### PR TITLE
F #2453:  Catch monitor errors

### PR DESCRIPTION
**Pull request by sergiojvg**

### CODE REVIEW

**Comment by sergiojvg** [comment]
--> master

File: src/vmm_mad/remotes/lib/vcenter_driver/host.rb [hunk 1]
```diff
@@ -344,199 +344,203 @@ def monitor_host_systems
...
-            str_info << vm_info
-        end
+            view.DestroyView # Destroy the view
 
-        view.DestroyView # Destroy the view
-
-        return str_info, last_mon_time
+            return str_info, last_mon_time
+        
+        rescue
```
Comment (tinova) [inline src/vmm_mad/remotes/lib/vcenter_driver/host.rb]: Needs to rescue only StandardError

File: src/vmm_mad/remotes/lib/vcenter_driver/host.rb [hunk 2]
```diff
@@ -344,199 +344,203 @@ def monitor_host_systems
...
+            return "", ""
```
Comment (tinova) [inline src/vmm_mad/remotes/lib/vcenter_driver/host.rb]: Shouldn't it add error_monitoring as well?

**Comment by tinova** [comment]
not needed, already solved #2453

**Pull request closed without merging**